reactDashboard/src/pages/Overview.jsx

```diff
@@ -29,10 +29,13 @@ const Overview = ({
   const [areaType, setAreaType] = useState(AREATYPE_HEALTH_BOARDS);
   const [valueType, setValueType] = useState(VALUETYPE_CASES);
   const [chartType, setChartType] = useState(DAILY_CASES);
-  const [zoomDataCharts, setZoomDataCharts] = useState(false);
+  const [dateRange, setDateRange] = useState({ startDate: 0, endDate: 0 });
   const [zoomGeoMap, setZoomGeoMap] = useState(false);
+  const [maxDateRange, setMaxDateRange] = useState({
```

Comment:
Same comments about pushing state and logic down into the dataChart component as made in Regional.jsx

Reply:
moved to separate ticket